examples/graphql-cache-invalidation/graphql_cache_invalidation.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Cache Invalidation
+
+The Ballerina `graphql` module provides functionality for cache invalidation. The `invalidate(resourcePath: string)` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a `graphql:Service`. To invalidate a specific cache in a `resource` function, the `invalidate(resourcePath: string)` API requires to specify the name of the resource. On the other hand, the `invalidateAll()` API invalidates all caches within the service.
```

Comment:
`invalidate(resourcePath: string)` doesn't make sense. It is a Ballerina API, but doesn't look like one here.